src/ERC721Drop.sol

```diff
@@ -1267,6 +1272,38 @@ contract ERC721Drop is
         emit MintFeePayout(zoraFee, ZORA_MINT_FEE_RECIPIENT, success);
     }
 
+    function _requireCanMintQuantity(uint256 quantity) internal view {
+        if (quantity + _totalMinted() > config.editionSize) {
+            revert Mint_SoldOut();
+        }
+    }
+
+    function _handleSupplyRoyalty(uint256 mintAmount) internal returns (uint256 totalRoyaltyMints) {
+        if (royaltyMintSchedule == 0) {
+            // If we still have no schedule, return 0 supply royalty.
+            return 0;
+        }
+
+        totalRoyaltyMints = (mintAmount + (_totalMinted() % royaltyMintSchedule)) / (royaltyMintSchedule - 1);
+
+        if (totalRoyaltyMints > 0) {
+            address royaltyRecipient = config.fundsRecipient;
+
+            // If we have no recipient set, return 0 supply royalty.
+            if (royaltyRecipient == address(0)) {
+                return 0;
+            }
+            _mintNFTs(royaltyRecipient, totalRoyaltyMints);
+        }
+    }
+
+    function updateRoyaltyMintSchedule(uint32 newSchedule) external onlyAdmin {
+        if (newSchedule == 1) {
```

Comment:
why this error? What's the `1` represent?

Reply:
this would mean every single mint goes to creator

Reply:
not quite sure I understand what the `1` means.  could you make the `1` an enum, or have it be a constant with a name explaining what `1` represents?

Reply:
ok i get it now - schedule is how many mints would result in a royalty mint.  I'd add a comment here explaining why this is not allowed